docs/framework/install/on-server-2025.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Install .NET Framework on Windows Server 2025
+description: Learn how to install .NET Framework on Windows Server 2025.
+ms.date: 11/21/2024
```

Comment:
should this a more recent date since you haven't shipped yet? other articles touched should also have their dates updated

```suggestion
ms.date: 12/05/2024
```